vms/platformvm/txs/mempool/mempooltest/mempool.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package mempooltest
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/utils/lock"
+	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs/mempool"
+)
+
+var _ mempool.Mempool = (*FakeMempool)(nil)
+
+// FakeMempool performs no verification and has no upper bound on txs
+type FakeMempool struct {
```

Comment:
`mempooltest.FakeMempool` seems a bit redundant. Should this just be:
```suggestion
type Mempool struct {
```
?

Reply:
Yeah - renamed.